dockers/docker-platform-monitor/supervisord.conf

```diff
@@ -33,9 +33,11 @@ startsecs=0
 command=/usr/sbin/fancontrol
 priority=4
 autostart=false
-autorestart=false
+autorestart=unexpected
+stopwaitsecs=10
```

Comment:
10 seconds is the default value in supervisor, thus this line is unnecessary. Also, what does `stopwaitsecs` have to do with restarting the process?

Reply:
@jleveque we can remove this line. I thought the default is 1. I want add some delay before killing the process to add some delay to add delay between process kill and process restart. But startsecs should be enough for that case, I think.

Reply:
I don't think `stopwaitsecs` is the parameter you're looking for. This configures the amount of time supervisor will wait for the process to exit before sending a SIGKILL:

```
stopwaitsecs

The number of seconds to wait for the OS to return a SIGCHLD to supervisord after the program has been sent a stopsignal. If this number of seconds elapses before supervisord receives a SIGCHLD from the process, supervisord will attempt to kill it with a final SIGKILL.

Default: 10

Required: No.

Introduced: 3.0
```